recipes/tesseract/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+./autogen.sh
```

Comment:
Normally people run `autoreconf -fi`, which does this and some other things. Also it appears there is a `CMakeList.txt`. So you could use `cmake` instead if you rather. The latter is a bit easier IMHO to use on Windows, but works fine on Unix too.